serverless/serverless
# Add integration test for deploying with --noDeploy flag

## What did you implement:



## How did you implement it:



## How can we verify it:

Running integration test `tests/integration/aws/general/deployment/tests.js`should first create package with `--noDeploy` flag, then check that CloudFormation templates and zip that contains functions exists and at last check that there is no CloudFormation stack for service in AWS.




## Todos:



***Is this ready for review?:*** YES
